goodRuns/setup.sh

```
@@ -0,0 +1,4 @@
+#!/bin/bash
+source /opt/sphenix/core/bin/sphenix_setup.sh -n new
+export RUNLISTGEN_SRC="$(pwd)"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)